### DISCUSSION TITLE
A question about the aimd calcuation by VASP

### DISCUSSION THREAD

**Discussion by LZYUCL** [deepmodeling/dpgen#1174] [Q&A]
When Usually, when we use the AIMD process of the vasp calculation system, a key parameter **SMASS** will be involved. I will first set this parameter to **0** in the general calculation process, and then adjust this parameter according to the SMASS output in the vasprun file. But the SMASS in INCAR_md I saw in the dpgen example is set to 0. I am not sure whether this value will be changed according to the output of vasp in the dpgen cycle. **Will it affect the accuracy of the result if it is always set to 0?**

**Comment by PolyuWeldingSpock** [comment]
In my opinion, examples of DPGEN just provided some reference not accurate parameter especially in fp steps. You should set your own parameter according to your own experience .
